### PR TITLE
Change the use of the word eject everywhere to remove in the main page

### PR DESCRIPTION
**Pull request by MatthewDaggitt**
Fulfilling the change requested by the client Taylor.

Little unsure about the choice of the word "remove". Maybe there's a better synonym?

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+function removeStudent() {
   // Update the database
-  numberOfStudentsEjected += 1;
-  displayNumberOfStudentsEjected();
+  numberOfStudentsRemove += 1;
+  displayNumberOfStudentsRemoved();
 }
 
-function displayNumberOfStudentsEjected() {
+function displayNumberOfStudentsRemoved() {
```
Comment (ghost) [inline test.js]: Maybe we should take this opportunity to change the word "display" as well?

File: test.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
   alert("Hi3");
 }
 
-function ejectStudent() {
+function removeStudent() {
   // Update the database
-  numberOfStudentsEjected += 1;
-  displayNumberOfStudentsEjected();
+  numberOfStudentsRemove += 1;
+  displayNumberOfStudentsRemoved();
```
Comment (yunho7687) [inline test.js]: test(pls ignore me)

Comment (MatthewDaggitt) [reply test.js]: Ah yes, this feature would be great, but way out of scope for this PR. Consider yourself ignored.

File: test.js (hunk 1 above)
Comment (MatthewDaggitt) [reply test.js]: Yes, great idea. Changed.

**Review by ghost — APPROVED** [review]

**Pull request merged**